support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -172,6 +178,48 @@ function campaignSpecificDetails() {
   return {};
 }
 
+function goalReachedTemplate() {
+  if (isFrontlineCampaign()) {
+    return (
+      <div className="goal-reached">
+        <div className="goal-reached__message">
+          Thank you to everyone who supported ‘The Frontline’.
+          We’re no longer accepting contributions for the series, but you can still support
+          The Guardian’s journalism with a single or recurring contribution
+        </div>
```

Comment:
Text should live in `p`s or `span`s in a perfect world but this won't break anything